README.md

```diff
@@ -26,25 +26,39 @@ I didn't come up with all the rules out of nowhere - they are mostly
 based on my extensive career as a professional software engineer,
 feedback and suggestions from members of the Ruby community and
 various highly regarded Ruby programming resources, such as
-["Programming Ruby 1.9"][pickaxe] and
+["Programming Ruby"][pickaxe] and
 ["The Ruby Programming Language"][trpl].
 
 You can generate a PDF or an HTML copy of this guide using
-[Transmuter][].
+[Pandoc][].
 
 [RuboCop][] is a code analyzer, based on this
 style guide.
 
+Translations of the guide are available in the following languages:
```

Comment:
It will keep conflicting if we do. I think we may have already stripped it and I resolved by keeping it this time instead of re-removing.